### PR TITLE
Adds create png function

### PR DESCRIPTION
**Pull request by AnturK**
To go around the painting problem and maybe other things in the future.

### CODE REVIEW

File: src/dmi.rs [hunk 1]
```diff
@@ -28,3 +31,25 @@ fn write_png(path: &str, info: OutputInfo, image: Vec<u8>) -> Result<()> {
...
     Ok(writer.write_image_data(&image)?)
 }
+
+fn create_png(path: &str, width: &str, height: &str, data: &str) -> Result<()> {  
+    let width = u32::from_str_radix(width,10)?;
+    let height = u32::from_str_radix(height,10)?;
+    
+    let mut result : Vec<u8> = Vec::new();
+    for chunk in data.chars().collect::<Vec<char>>().chunks(7) {
+        let single: String = chunk.into_iter().collect();
```
Comment (SpaceManiac) [inline src/dmi.rs]: I feel like you could do this without as much `collect()`

Comment (AnturK) [reply src/dmi.rs]: Alright, changed it to SO suggestion, i feel getting rid of inner collect would make this bit unreadable.

**Comment by SpaceManiac** [comment]
do you have some test data for this so I could do some cleanups without accidentally breaking anything?

also how soon do you need binaries with this included?

**Comment by AnturK** [comment]
Scrounged some from my persistence tests.

19x19

```
#000000#DA0000#DA0000#DA0000#DA0000#DA0000#DA0000#DA0000#DA0000#DA0000#DA0000#DA0000#DA0000#DA0000#DA0000#DA0000#DA0000#DA0000#000000#ffffff#000000#DA0000#DA0000#DA0000#DA0000#DA0000#DA0000#DA0000#DA0000#DA0000#DA0000#DA0000#DA0000#DA0000#DA0000#DA0000#000000#ffffff#ffffff#ffffff#000000#DA0000#DA0000#DA0000#DA0000#DA0000#DA0000#DA0000#DA0000#DA0000#DA0000#DA0000#DA0000#DA0000#000000#ffffff#ffffff#ffffff#ffffff#ffffff#000000#DA0000#DA0000#DA0000#DA0000#DA0000#DA0000#DA0000#DA0000#DA0000#DA0000#DA0000#000000#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#000000#000000#000000#000000#000000#000000#000000#000000#000000#000000#000000#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#000000#000000#000000#000000#000000#000000#000000#000000#000000#000000#000000#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#000000#A8E61D#A8E61D#A8E61D#A8E61D#A8E61D#A8E61D#A8E61D#A8E61D#A8E61D#000000#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#000000#A8E61D#000000#000000#000000#A8E61D#000000#A8E61D#000000#A8E61D#000000#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#000000#A8E61D#000000#A8E61D#000000#A8E61D#000000#A8E61D#000000#A8E61D#000000#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#000000#A8E61D#000000#A8E61D#000000#A8E61D#000000#000000#A8E61D#A8E61D#000000#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#000000#A8E61D#000000#A8E61D#000000#A8E61D#000000#A8E61D#000000#A8E61D#000000#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#000000#A8E61D#000000#000000#000000#A8E61D#000000#A8E61D#000000#A8E61D#000000#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#000000#A8E61D#A8E61D#A8E61D#A8E61D#A8E61D#A8E61D#A8E61D#A8E61D#A8E61D#000000#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#000000#000000#000000#000000#000000#000000#000000#000000#000000#000000#000000#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#000000#000000#000000#000000#000000#000000#000000#000000#000000#000000#000000#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#000000#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#000000#ffffff#ffffff#ffffff#ffffff#ffffff#000000#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#000000#ffffff#ffffff#ffffff#000000#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#000000#ffffff#000000#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#000000
```

23x19
```
#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#DA0000#ffffff#ffffff#ffffff#FFF200#ffffff#ffffff#FFF200#ffffff#ffffff#ffffff#DA00FF#ffffff#ffffff#DA00FF#ffffff#ffffff#DA00FF#ffffff#ffffff#ffffff#ffffff#ffffff#DA0000#ffffff#ffffff#ffffff#FFF200#ffffff#ffffff#FFF200#ffffff#ffffff#ffffff#DA00FF#ffffff#ffffff#DA00FF#ffffff#ffffff#DA00FF#ffffff#ffffff#ffffff#ffffff#ffffff#DA0000#ffffff#ffffff#ffffff#FFF200#ffffff#ffffff#FFF200#ffffff#ffffff#ffffff#DA00FF#ffffff#ffffff#DA00FF#ffffff#ffffff#DA00FF#ffffff#ffffff#ffffff#ffffff#ffffff#DA0000#ffffff#ffffff#ffffff#FFF200#ffffff#ffffff#FFF200#ffffff#ffffff#ffffff#DA00FF#ffffff#ffffff#DA00FF#ffffff#ffffff#DA00FF#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#A8E61D#ffffff#A8E61D#ffffff#ffffff#ffffff#A8E61D#ffffff#ffffff#ffffff#00B7EF#ffffff#ffffff#ffffff#00B7EF#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#A8E61D#ffffff#A8E61D#ffffff#ffffff#ffffff#A8E61D#ffffff#ffffff#ffffff#00B7EF#ffffff#ffffff#ffffff#00B7EF#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#A8E61D#ffffff#ffffff#A8E61D#ffffff#A8E61D#ffffff#ffffff#ffffff#ffffff#ffffff#00B7EF#ffffff#00B7EF#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#A8E61D#ffffff#ffffff#A8E61D#ffffff#A8E61D#ffffff#ffffff#ffffff#ffffff#ffffff#00B7EF#ffffff#00B7EF#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#A8E61D#ffffff#ffffff#ffffff#A8E61D#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#00B7EF#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#FF9300#ffffff#ffffff#ffffff#FF9300#ffffff#ffffff#ffffff#FF9300#ffffff#ffffff#ffffff#FF9300#ffffff#ffffff#ffffff#FF9300#ffffff#ffffff#ffffff#FF9300#ffffff#ffffff#ffffff#FF9300#ffffff#FF9300#ffffff#FF9300#ffffff#FF9300#ffffff#FF9300#ffffff#FF9300#ffffff#FF9300#ffffff#FF9300#ffffff#FF9300#ffffff#FF9300#ffffff#FF9300#ffffff#ffffff#ffffff#FF9300#ffffff#ffffff#ffffff#FF9300#ffffff#ffffff#ffffff#FF9300#ffffff#ffffff#ffffff#FF9300#ffffff#ffffff#ffffff#FF9300#ffffff#ffffff#ffffff#FF9300#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff#ffffff
```

Binaries ideally when it works, but this is feature enabler not anything urgent so no hurry.

**Comment by SpaceManiac** [comment]
if that looks good to you then I'll merge

**Comment by AnturK** [comment]
Looks good.

**Pull request merged**